modules/core/src/v2/trading/affirmations.ts

```diff
@@ -39,16 +50,28 @@ export class Affirmations {
   /**
    * Retrieves a single affirmation by its ID
    * @param id ID of the affirmation to retrieve
+   * @param accountId ID of the trading account that the affirmation belongs to
    * @param callback
    */
-  get({ id }, callback): Bluebird<Affirmation> {
+  get({ id, accountId }: GetAffirmationParameters, callback?: NodeCallback<Affirmation>): Bluebird<Affirmation> {
     return co(function* get() {
-      const url = this.bitgo.microservicesUrl(`/api/trade/v1/enterprise/${this.enterprise.id}/affirmations/${id}`);
+      if (!this.account && !accountId) {
+        throw new Error('accountId must be provided in parameters for an enterprise context');
+      }
+
+      const url = this.bitgo.microservicesUrl(
+        `/api/trade/v1/enterprise/${this.enterpriseId}/account/${accountId || this.account.id}/affirmations/${id}`
+      );
       const response = yield this.bitgo.get(url).result();
 
-      return new Affirmation(response, this.bitgo);
+      return new Affirmation(response, this.bitgo, this.enterpriseId);
     })
       .call(this)
       .asCallback(callback);
   }
 }
+
+interface GetAffirmationParameters {
```

Comment:
Also if you're going to make more changes, this should go above the class definition.

Reply:
done, and re-ordered all other interfaces in the trade folder